indra_db_service/api.py

```diff
@@ -193,42 +194,33 @@ def search():
         vue_style=vue_style, stmt_types_json=stmt_types_json)
 
 
+suf_ct_map = {'.js': 'application/javascript', '.css': 'text/css'}
+
+
 @app.route('/data-vis/<path:file_path>')
 def serve_data_vis(file_path):
-    full_path = path.join(HERE, 'data-vis/dist', file_path)
+    full_path = HERE / 'data-vis' / 'dist' / file_path
```

Comment:
This might be the most legible:

```suggestion
    full_path = HERE.joinpath('data-vis', 'dist', file_path)
```

Reply:
Personally, I disagree. I think "/" is well established as the divider between path segments, making the statement unambiguous. And someone looking at it that doesn't "know" programming would in fact be less confused and have an easier time interpreting it, which to my mind makes it more "pythonic".

That said, it is pretty subjective, and I am most certainly open to rebuttals :)

Reply:
I don't have a strong opinion. I think there's no mistaking what's going on with the slashes